index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bhekisile's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <nav class="nav_bar">
+                <span class="start-nav">My Logo</span>
+                <span class="end-nav"><a href="#">Contact</a></span>
+        </nav>
+    <div class="img_background">
+    <main>
+        <section>
+            <h1>I’m Marian<br> 
+                Glad to see<br> you!</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website
+                 Look through some of my work and experience! If you like what you see and have a 
+                 project you need coded, don’t hestiate to contact me.</p>
+        </section>
+    </main>
+    <section>
+        <div class="connect">LET’S CONNECT</div>
+         <div class="social">
+            <ul>
+                <li><img class="twitter" src="image/twitter.svg" alt="twitter"></li>
+                <li><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></li>
+                <li><img class="medium" src="image/medium.svg" alt="Medium"></li>
+                <li><img class="github" src="image/github.svg" alt="GitHub"></li>
+                <li><img class="angellist" src="image/angellist.svg" alt="Angellist"></li>
+            </ul>
+         </div>
+    </section>
```

Comment:
Good job! but all of this is just a single section, that's why you may not need to have multiple sections. And here is what to improve.

1. It's required to follow the exact Figma design typography, colors, and sizes, so please make sure to update the font size of your header tag, and also use the exact font for your text. This will make your design look much more beautiful.
2. The `LET'S CONNECT` is supposed to be a link here.
3. Your icons look fantastic, but as they're linked to your social media accounts it's recommended to put the `img` tags into `a` tags.

Yours:

![image](https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png)


Original:

![image](https://user-images.githubusercontent.com/34779928/205124237-42f15b94-4206-422f-9fd0-1e6bab7a6d16.png)